lib/Yield/CMakeLists.txt

```diff
@@ -12,7 +12,6 @@ revng_add_library_internal(
   ControlFlow/Extraction.cpp
   ControlFlow/FallthroughDetection.cpp
   ControlFlow/NodeSizeCalculation.cpp
-  ControlFlow/SVG.cpp
```

Comment:
I cherry-picked it because I wasn't sure which one will go in first and I wouldn't want to do a painful rebase because of that.

It's just a file being moved.